### PR TITLE
Add the selected solution in evaluation design

### PR DESCRIPTION
**Pull request by workingloong**
Though we have discussed the pros and cons of several proposal solutions, we don't select which solution to implement.

### CODE REVIEW

File: docs/designs/evaluation_design.md [hunk 1]
```diff
@@ -23,39 +23,50 @@ As shown in the 10th step of <em> Figure 1.</em>, the master generates evaluatio
...
 
-2. Existing pservers stop train task and load model from checkpoint to evaluate.\
-When an evaluation job starts, pservers save the current model to the temporary file and load model from the last checkpoint. After evaluation, pservers will restore the model from temporary file preserved and continue to train. The design of pserver will be complicated in this case. What's more, some workers may be reporting gradients when the master change model to evaluation for pservers and pservers can not update model by those gradients at this moment. So, this method may be not good choice for high cost.
+2. The existing PS stop training tasks and load model from checkpoint to evaluate.\
```
Comment (skydoorkai) [inline docs/designs/evaluation_design.md]: stop, load -> stops, loads.
from a checkpoint

File: docs/designs/evaluation_design.md [hunk 2]
```diff
@@ -23,39 +23,50 @@ As shown in the 10th step of <em> Figure 1.</em>, the master generates evaluatio
...
+When an evaluation job starts, the PS save the current model to the temporary file and load model from the last checkpoint. After evaluation, PS will restore the model from temporary file preserved and continue to train. The design of PS will be very complicated in this case. What's more, some workers may be reporting gradients when the master change model to evaluation for PS and PS can not update model by those gradients at this moment. So, this method may be not good choice for high cost.
```
Comment (skydoorkai) [inline docs/designs/evaluation_design.md]: save -> saves, also loads.

Comment (workingloong) [reply docs/designs/evaluation_design.md]: Done

File: docs/designs/evaluation_design.md (hunk 1 above)
Comment (workingloong) [reply docs/designs/evaluation_design.md]: Done

File: docs/designs/evaluation_design.md (hunk 2 above)
Comment (workingloong) [reply docs/designs/evaluation_design.md]: Done

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**